src/components/Discussion/Internal/Comment/Header.js

```diff
@@ -294,59 +309,40 @@ export const Header = ({ t, comment, menu, isExpanded, onToggle }) => {
           )}
         </div>
       </div>
-      {onToggle && (
-        <button
-          {...headerActionStyle({ isExpanded })}
-          {...headerActionStyleHover}
-          {...colorScheme.set('color', isExpanded ? 'divider' : 'textSoft')}
-          onClick={onToggle}
-        >
-          {!isExpanded && comments && comments.totalCount > 0 && (
-            <div {...styles.expandCount}>
-              {t.pluralize('styleguide/comment/header/expandCount', {
-                count: comments.totalCount + 1
-              })}
-            </div>
-          )}
-          {isExpanded ? <IcCollapse /> : <IcExpand />}
-        </button>
-      )}
-      {menu && (
-        <CalloutMenu Element={MoreIconWithProps} align='right'>
-          {menu}
-        </CalloutMenu>
-      )}
+      <div {...styles.actionsWrapper}>
+        {onToggle && (
+          <button
```

Comment:
Since the label is on the left instead of the right and custom styling is provided, this would cause quite a lot of refactoring. Also it would require making the IconButton even more complex. Therefore I'd rather not change this.

Reply:
There is no label on the expand button, nor is there a label on the more button, so I don't see an issue with using the default IconButton.

Reply:
There is a label left of the icon showing how many replies there are.

Reply:
![image](https://user-images.githubusercontent.com/30313631/142639148-934a1552-0709-4d8e-a05b-20a438582b3a.png)

Reply:
omg what is this nonesense XD (not on you :D)

Reply:
Done ✔️